### ISSUE TITLE
Examples: Language of page content is not indicated

### ISSUE THREAD

**Issue by TetraLogicalHelpdesk** [ampproject/amphtml#31172]
## WCAG Level
Level A / AA

## Priority
Low

## Type
Structure

## Pages/screens/components affected
* [amp-pixel](https://amp.dev/documentation/components/amp-pixel/)
* [amp-sidebar](https://amp.dev/documentation/components/amp-sidebar/)
* [amp-accordion](https://amp.dev/documentation/components/amp-accordion/)
* [amp-selector](https://amp.dev/documentation/components/amp-selector/)
* [amp-sticky-ad](https://amp.dev/documentation/components/amp-sticky-ad/)
* [amp-user-notification](https://amp.dev/documentation/components/amp-user-notification/)
* [amp-consent](https://amp.dev/documentation/components/amp-consent/)
* [amp-video](https://amp.dev/documentation/components/amp-video/)
* [amp-app-banner](https://amp.dev/documentation/components/amp-app-banner/)
* [amp-youtube](https://amp.dev/documentation/components/amp-youtube/)
* [amp-base-carousel](https://amp.dev/documentation/components/amp-base-carousel/)
* [amp-stream-gallery](https://amp.dev/documentation/components/amp-stream-gallery/)
* [amp-inline-gallery](https://amp.dev/documentation/components/amp-inline-gallery/)
* [amp-lightbox-gallery](https://amp.dev/documentation/components/amp-lightbox-gallery/)

## Description
It is important to programmatically identify the human language of a web page/section of a page, as this makes a difference as to how screen readers pronounce content.

We found pages that lack a programmatically defined language, which fails WCAG 2.1 SC 3.1.1 Language of the Page. This leads to the language of all *parts* of the content also lacking a programmatically defined language, failing WCAG 2.1 SC 3.1.2 Language of Parts as well.

## User impact
Depending on the user’s setup, screen readers and other text-to-speech applications that support multiple languages will not be able to adapt to the correct accent or pronunciation when announcing page content, which can impact on the understandability of the page.

## Required solution
Make sure that the human language of all HTML pages, as well as content embedded within pages and changes in language, is declared correctly. Make sure that the `<html>` element has the required `lang` attribute, and that the value of this attribute matches the human language of the page.

This solution must be applied to all instances of the issue identified within the test sample, then applied to all other instances of the same issue identified throughout the rest of the website.

## Test procedure(s)
Use these steps to confirm that the solution has been correctly applied:
1. Check that the `<html>` element has a `lang` attribute.
2. Check that the value of the `lang` attribute reflects the primary language used by the page.

## Related standards
* [WCAG 2.1 Success Criterion 3.1.1 Language of Page (Level A)](https://www.w3.org/TR/WCAG21/#language-of-page)
* [WCAG 2.1 Success Criterion 3.1.2 Language of Parts (Level AA)](https://www.w3.org/TR/WCAG21/#language-of-parts)

## More information
* [W3C - Technique H57: Using the language attribute on the HTML element](https://www.w3.org/WAI/WCAG21/Techniques/html/H57.html)
* [W3C - Technique H58: Using language attributes to identify changes in the human language](https://www.w3.org/WAI/WCAG21/Techniques/html/H58.html)

## Test data
Test date: November 2020
